Pester.psm1

```diff
@@ -836,13 +836,17 @@ any code coverage information, because it's not supported by the schema.
 
 Enter the path to the files of code under test (not the test file).
 Wildcard characters are supported. If you omit the path, the default is local
-directory, not the directory specified by the Script parameter.
+directory, not the directory specified by the Script parameter. Pester test files
```

Comment:
This is a bit hard to understand when you don't know the internals. I would reword it along the lines of:  Pester test files are by default excluded from code coverage when a directory is provided. But when you provide a test file directly, code coverage will be measured. To include tests  in code coverage of a directory, use the dictionary syntax and provide IncludeTests = $true option, as shown below.

Reply:
I expected users to understand it since the first line in the help mentions _strings_ or hash table as valid input. I agree that it can be improved, but I'm unsure about this:

> When you provide a test file directly, code coverage will be measured.

I might assume that `@{ Path = "file.tests.ps1" }` would also include tests, which it won't when hashtables are used. Should it? I consider the dictionary as advanced input where I expect it to be set if needed. How about this?

> Pester test files are by default excluded from code coverage when a directory is provided. When you provide a test file directly using string, code coverage will be measured. To include tests in code coverage of a directory, use the dictionary syntax and provide IncludeTests = $true option, as shown below.

Reply:
Yup, agree.

Reply:
Updated help